### DISCUSSION TITLE
I'm trying to disable a row click on a relation manager but it's not working

### DISCUSSION THREAD

**Discussion by maxacarvalho** [filamentphp/filament#8142] [Help] [answered]
Hi,

I have a relation manager and I'd like to disable the row click to edit the record. I'm using:
```php
public function table(Table $table): Table
{
    return $table
        ->recordUrl(fn () => null)
...
```

But it does not work, the column continues to be clickable.
Ps.: Bear in mind that I still want the "EditAction" to work, but only when the user explicitly clicks on the action itself.

Thanks!

**Comment by leandrocfe** [comment, marked as answer]
```php
->recordAction(null)
```

**Reply by maxacarvalho** [reply]
That works, thanks! I couldn't find that method in the docs, only the `recordUrl` one.

**Reply by wanderfranca** [reply]
Valeu Leandro, funcionou aqui, obrigado!!!

**Reply by kangbobi** [reply]
return $table->recordUrl(null)
